docs/source/user_guide/installation/install-from-source.rst

```diff
@@ -46,7 +46,7 @@ You can install the above with
 
 	.. code:: bash
 
-		brew install python openblas gcc gfortran graphviz libomp cmake
+		brew install python graphviz libomp cmake
```

Comment:
Actually, we need these here, can you revert this change? These are pre-installed on macOS GitHub runners, but users might not have them.

Reply:
```suggestion
		brew install python openblas gcc gfortran graphviz libomp cmake
```

Reply:
Done